katran/lib/tests/KatranLbTest.cpp

```diff
@@ -24,34 +24,30 @@ namespace katran {
 class KatranLbTest : public ::testing::Test {
  protected:
   KatranLbTest()
-      : lb(KatranConfig{
-            "eth0", // mainInterface
-            "ipip0", // v4TunInterface
-            "ipip60", // v6TunInterface
-            "./lb.o", // balancerProgPath
-            "./lb_with_introspection.o", // balancerProgWithIntrospectionPath
-            "./hc.0", // healthcheckingProgPath
-            {0x00, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E}, // defaultMac
-            1, // priority
-            "", // rootMapPath
-            1, // rootMapPos
-            true, // enableHc
-            false, // disableForwarding
-            512, // maxVips
-            4096, // maxReals
-            65537, // chRingSize
-            true, // testing
-            1, // LruSize
-            {}, // forwardingCores
-            {}, // numaNodes
-            10, // maxLpmSrcSize
-            4, // maxDecapDst
-            "eth0", // hcInterface
-            0, // xdpAttachFlags
-            {}, // monitorConfig
-            false, // memlockUnlimited
-            {}, // localMac
-        }) {}
+      : lb(KatranConfig{"eth0",
+                        "ipip0",
```

Comment:
why leave out comments?

Reply:
this was revert of previous diff. which added comments as well.